docs/05-ingest.md

```diff
@@ -203,8 +203,8 @@ part of the job specification:
 
 ```properties
 sleeper.default.bulk.import.emr.release.label=emr-6.10.0 # The EMR release label to be used when creating an EMR cluster for bulk importing data using Spark running on EMR. This default can be overridden by a table property or by a property in the bulk import job specification.
-sleeper.default.bulk.import.emr.master.instance.types=m5.xlarge # The EC2 instance types to be used for the master node of the EMR cluster.
-sleeper.default.bulk.import.emr.executor.instance.types=m5.4xlarge # The EC2 instance types to be used for the executor nodes of the EMR cluster.
+sleeper.default.bulk.import.emr.master.x86.instance.types=m5.xlarge # The EC2 x86_64 instance types to be used for the master node of the EMR cluster.
+sleeper.default.bulk.import.emr.executor.x86.instance.types=m5.4xlarge # The EC2 x86_64 instance types to be used for the executor nodes of the EMR cluster.
 sleeper.default.bulk.import.emr.executor.initial.instances=2 # The initial number of EC2 instances to be used as executors in the EMR cluster.
 sleeper.default.bulk.import.emr.executor.max.instances=10 # The maximum number of EC2 instances to be used as executors in the EMR cluster.
```

Comment:
These property descriptions are out of date, as these are capacity units now rather than a number of instances.